internal/fields/sql.go

```diff
@@ -77,6 +77,10 @@ func (f Valuer) Value() (driver.Value, error) {
 			return iface.MarshalJSON()
 		}
 		return json.Marshal(i)
+	case f.Tags.Contains("implicitnull"):
```

Comment:
nit: what is implicit about this null? i kind of like `zeroisnull`, but i could be convinced otherwise

Reply:
No strong feelings either way, @berfarah @stephen whichever preference you guys want.